README.md

```diff
@@ -1 +1,45 @@
-# spot-electron-sdk
\ No newline at end of file
+# Spot Beacon Desktop SDK
+This SDK provides spot beacon functionality for desktop (electron) apps.
+
+# Usage
+1. Instantiate the SDK with (optional) configuration object.
+
+```
+const { BeaconEmitter } =  require('spot-electron-sdk');
+const emitter = new BeaconEmitter({
```

Comment:
I know it's an event emitter, and that it "emits" becaons, but wouldn't it be more accurate to call this a `BeaconDetector` ?